### DISCUSSION TITLE
Rollback because of same package name in internal and external npm registry

### DISCUSSION THREAD

**Discussion by kumarrishav** [renovatebot/renovate#15207] [Request Help]
### How are you running Renovate?

Self-hosted

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

Rollback PR because of the same package name in the internal and external npm registry.

We are seeing some rollback PR, where it creates PR to roll back to the public version a pkg. We have pkg with the same name in the internal registry as well. 

Expectation is to use internal registry as a source for update instead of external

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste any log here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by rarkins** [comment]
What you're describing sounds like a configuration problem and not a bug.

**Reply by kumarrishav** [reply]
Thanks for your response. can you point me to the right configuration for this? I.E It should use an internal registry over external for the pkg with the same name. For other private package - it does work with internal registry

issue comes when there is same package name exist in internal and external registry

**Reply by rarkins** [reply]
No, you haven't provided enough information for us to help you further. Are you familiar with npm concepts such as `.npmrc` files?
